src/cmdlib.sh

```diff
@@ -429,7 +429,8 @@ runvm() {
 
     # tmp_builddir is set in prepare_build, but some stages may not
     # know that it exists.
```

Comment:
Looking at the code a bit closer should we just remedy this ^^ so that we don't have to set `$tmp_builddir` twice (once here and once in `prepare_build`)?